### ISSUE TITLE
While reading, `cellparts` are not being reset at end of cell tag.

### ISSUE THREAD

**Issue by strichter** [bradbase/xlcalculator#10]
You need to add the foillowing line to `reader.py` line 331:

```
cellparts = {'address' : None, 'value': None, 'formula': None, 'ref' : None}
```

Otherwise previous values will leak into different cells. That happens on all Openpyxl generated spreadsheets with values since you are not handling `inlineStr` types correctly.

**Comment by bradbase** [comment]
I can see I'm not re-setting the cellparts correctly.

It obviously needs to happen on an 'end' column event, but I think it needs to happen after the cell value gets set.

From what I can see that line might be better on line 332 as that reset needs to happen whether there's a formula or not.

Is that adequate from your perspective?

**Comment by bradbase** [comment]
Have released version 0.0.7b

**Comment by strichter** [comment]
Yeah, I was guestimating, since I had other changes in the file.
